capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -991,9 +1001,9 @@ private void importDocumentFromIntent(@NonNull final Intent data) {
                 } else {
                     final FileImportValidator.Error error = fileImportValidator.getError();
                     if (error != null) {
-                        showInvalidFileError(error);
-                    } else {
-                        showGenericInvalidFileError();
+                        Error errorClass = new Error(FileImportValidator.Error.valueOf(error.name()));
```

Comment:
[Request] Change in the requirements: please show a dialog using the `ErrorType`'s title string. Analog to the `showMultiPageLimitError()` method.